### ISSUE TITLE
The extension stopped working after upgrading Visual Studio 2022 to the latest version 17.4

### ISSUE THREAD

**Issue by javicabanas** [fsprojects/fantomas-for-vs#40]
Try:

Uninstall the extension and installing again, getting this message:

> This extension is not installable on any currently installed products

Log:

```
16-Nov-22 4:56:48 PM - Microsoft VSIX Installer
16-Nov-22 4:56:48 PM - -------------------------------------------
16-Nov-22 4:56:48 PM - vsixinstaller.exe version:
16-Nov-22 4:56:48 PM - 17.4.2118-17-4P4Inse-ge4c88902
16-Nov-22 4:56:48 PM - -------------------------------------------
16-Nov-22 4:56:48 PM - Command line parameters:
16-Nov-22 4:56:48 PM - C:\Program Files (x86)\Microsoft Visual Studio\Installer\resources\app\ServiceHub\Services\Microsoft.VisualStudio.Setup.Service\VSIXInstaller.exe,C:\Users\fernanfr\Downloads\FantomasVs.vsix
16-Nov-22 4:56:48 PM - -------------------------------------------
16-Nov-22 4:56:48 PM - Microsoft VSIX Installer
16-Nov-22 4:56:48 PM - -------------------------------------------
16-Nov-22 4:56:48 PM - Skipping product Microsoft.VisualStudio.Product.BuildTools (ed51a08c) since it does not support extensions
16-Nov-22 4:56:48 PM - Initializing Install...
16-Nov-22 4:56:48 PM - Extension Details...
16-Nov-22 4:56:48 PM - 	Identifier         : FantomasVs.ef00bfc2-a899-45fc-aae8-afecf8673aaf
16-Nov-22 4:56:48 PM - 	Name               : F# Formatting
16-Nov-22 4:56:48 PM - 	Author             : Asti
16-Nov-22 4:56:48 PM - 	Version            : 1.0.2
16-Nov-22 4:56:48 PM - 	Description        : F# source code formatting using Fantomas. 
16-Nov-22 4:56:48 PM - 	Locale             : en-US
16-Nov-22 4:56:48 PM - 	MoreInfoURL        : https://github.com/deviousasti/fsharp-formatting-for-vs
16-Nov-22 4:56:48 PM - 	InstalledByMSI     : False
16-Nov-22 4:56:48 PM - 	SupportedFrameworkVersionRange : [4.5,)
16-Nov-22 4:56:48 PM - 
16-Nov-22 4:56:48 PM - 	SignatureState     : Unsigned
16-Nov-22 4:56:48 PM - 	Supported Products : 
16-Nov-22 4:56:48 PM - 		Microsoft.VisualStudio.Community
16-Nov-22 4:56:48 PM - 			Version : [16.0,17.0)
16-Nov-22 4:56:48 PM - 			ProductArchitecture : x86
16-Nov-22 4:56:48 PM - 
16-Nov-22 4:56:48 PM - 	References         : 
16-Nov-22 4:56:48 PM - 	Prerequisites      : 
16-Nov-22 4:56:48 PM - 		-------------------------------------------------------
16-Nov-22 4:56:48 PM - 		Identifier   : Microsoft.VisualStudio.Component.CoreEditor
16-Nov-22 4:56:48 PM - 		Name         : Visual Studio core editor
16-Nov-22 4:56:48 PM - 		Version      : [16.0,17.0)
16-Nov-22 4:56:48 PM - 
16-Nov-22 4:56:48 PM - Signature Details...
16-Nov-22 4:56:48 PM - 	Extension is not signed.
16-Nov-22 4:56:48 PM - 
16-Nov-22 4:56:48 PM - Searching for applicable products...
16-Nov-22 4:56:48 PM - Found installed product - Global Location
16-Nov-22 4:56:48 PM - Found installed product - Visual Studio Professional 2022
16-Nov-22 4:56:49 PM - VSIXInstaller.NoApplicableSKUsException: This extension is not installable on any currently installed products.
   at VSIXInstaller.ExtensionService.GetInstallableDataImpl(IInstallableExtension extension, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   at VSIXInstaller.ExtensionService.GetInstallableData(String vsixPath, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   at VSIXInstaller.ExtensionPackService.IsExtensionPack(IStateData stateData, Boolean isRepairSupported)
   at VSIXInstaller.ExtensionPackService.ExpandExtensionPackToInstall(IStateData stateData, Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize(Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize()
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
```

**Comment by deviousasti** [comment]
```
16-Nov-22 4:56:48 PM - 	Version            : 1.0.2
```
The latest published version is `1.1.0` - not sure how you're seeing this.
I'm still on 17.3, I'll upgrade to 17.4 and let you know. (RIP ME)

**Comment by javicabanas** [comment]
I have tried again and now worked fine, thanks.
